android/build.gradle

```diff
@@ -1,25 +1,22 @@
 buildscript {
     repositories {
         google()
-        maven {
-            url "https://maven.google.com"
-        }
-        jcenter()
```

Comment:
`jcenter()` is deprecated, migration to `mavenCentral()` is recommended https://developer.android.com/studio/build/jcenter-migration